src/main/java/net/atos/zac/aanvraag/ProductAanvraagService.java

```diff
@@ -79,6 +86,15 @@ public void verwerkProductAanvraag(final URI productAanvraagUrl) {
         zaak.setStartdatum(object.getRecord().getStartAt());
         zaak.setBronorganisatie(BRON_ORGANISATIE);
         zaak.setVerantwoordelijkeOrganisatie(BRON_ORGANISATIE);
+        try {
```

Comment:
Deze try catch kan weg, zie vorig comment.

Reply:
Opgelost in meest recente commit